Hotfixes teaser export

DaJansenGit opened the pull request:
This will be merged after #545

veronikarichter reviewed (changes requested):
Please fix TEASER pipeline

DaJansenGit commented:
> Please fix TEASER pipeline

Done. `development ` branch currently holds a bug that is fixed with this branch. So I updated the reference results.
Background:
I overwrite the values for the maximum heating and cooling power of the ideal heaters in the thermal zones for teaser via bim2sim in the teaser export. But they were reset before (in development) by an error of mine. Now the data is overwritten correctly. Hence the difference in the regression results. Since the new results are "more correct", I have updated them.
